packages/suite/src/components/suite/modals/metadata/MetadataProvider/index.tsx

```diff
@@ -57,11 +57,18 @@ const MetadataProvider = (props: Props) => {
     const connect = async (type: MetadataProviderType) => {
         setIsLoading(type);
         const result = await connectProvider(type);
-        if (!result) {
+        // window close indicates user action, user knows what happened, no need to show an error message
+        if (result === 'window closed') {
             setIsLoading('');
-            // TODO: error state, try again
+            // stop here, user might have changed his decision and wants to use another provider
             return;
         }
+        // TODO: we could show error in the UI
```

Comment:
i would rather implement this "display error", at least we got some feedback if it happens and we know where to look.
its always better to receive a issue with "it's showing Error: some error" than "it isn't working, nothing happens" :)

Reply:
oook, I just don't know how it should look like. I may try something and if design people don't like it, they will come with something

Reply:
Indeed it looks like it does not break anything and lets me remove this eslint ignore, but lets wait for tests..

Reply:
Maybe something like this will do for now
![image](https://user-images.githubusercontent.com/30367552/100106472-ddf17f00-2e68-11eb-9ff4-aaa411666a7e.png)

Reply:
But I am wondering whether this text should be displayed to user at all :face_with_head_bandage: It clerly seems to be addressing me rather than user :no_good: